### PR TITLE
Remove UDP Receiver

### PR DESCRIPTION
**Pull request by ZhangShenao**
## What is the purpose of the change

Remove `UdpReceiver` for client-v2.

## Brief changelog

1. Remove `UdpConnector.UdpReceiver` task to release udp port on client.
2. Remove udp request param in instance query api.
3. Remove Unnecessary mock stub in `UdpConnectorTest`.

### CODE REVIEW

**Comment by ZhangShenao** [comment]
#10824

File: client/src/main/java/com/alibaba/nacos/client/naming/remote/http/NamingHttpClientProxy.java [hunk 1]
```diff
@@ -212,7 +211,6 @@ public ServiceInfo queryInstancesOfService(String serviceName, String groupName,
         params.put(CommonParams.NAMESPACE_ID, namespaceId);
         params.put(CommonParams.SERVICE_NAME, NamingUtils.getGroupedName(serviceName, groupName));
         params.put(CLUSTERS_PARAM, clusters);
-        params.put(UDP_PORT_PARAM, String.valueOf(udpPort));
```
Comment (KomachiSion) [inline client/src/main/java/com/alibaba/nacos/client/naming/remote/http/NamingHttpClientProxy.java:215]: Http client request this port, if you need remove, please remove whole implementation, or keep this parameter.

File: naming/src/main/java/com/alibaba/nacos/naming/remote/udp/UdpConnector.java
Comment (KomachiSion) [inline naming/src/main/java/com/alibaba/nacos/naming/remote/udp/UdpConnector.java:1]: Why not remove the class derectly.

**Review by KomachiSion — CHANGES_REQUESTED** [review]

File: naming/src/main/java/com/alibaba/nacos/naming/remote/udp/UdpConnector.java
Comment (ZhangShenao) [reply naming/src/main/java/com/alibaba/nacos/naming/remote/udp/UdpConnector.java:1]: `UdpConnector` is still used by `UdpPushService`, and `PushExecutorUdpImpl` is an alternative implementation of `PushExecutorDelegate`. Will it cause significant impact if remove `UdpConnector` directly?

**Comment by KomachiSion** [comment]
OK.

**Review by KomachiSion — APPROVED** [review]

**Review by KomachiSion — CHANGES_REQUESTED** [review]
I see again this PR. I found that this UdpConnector is used by server part.

I think the Udpreceiver can't delete. because some 1.X client need it.

**Pull request closed without merging**